tools/inspect/inspect.cpp

```diff
@@ -1062,7 +1077,8 @@ void print_output(std::ostream& out, inspector_list const& inspectors)
       "<tr>\n"
       "<td>"
       "<a href = \"https://github.com/STEllAR-GROUP/hpx\">"
-      "<img src=\"http://stellar.cct.lsu.edu/files/stellar100.png\" alt=\"STE||AR logo\" />"
+      "<img src=\"http://stellar.cct.lsu.edu/files/stellar100.png\"\
+        alt=\"STE||AR logo\" />"
```

Comment:
Please don't break a quoted string in the middle by inserting a `\<newline>`. This is just confusing, Use compile time string concatination instead:

    "<img src=\"http://stellar.cct.lsu.edu/files/stellar100.png\""
        " alt=\"STE||AR logo\" />"